task-random-item.js

```diff
@@ -0,0 +1,8 @@
+//3. Napisz funkcję randomItem(), która zwróci losowy element podanej tablicy.
+
+var array = [1, 2, 3, 4, 5, 6];
+
+function randomItem(array) {
```

Comment:
Please move this function to the top of this file. That's a convention: first functions, then the rest.